MockDuck/Sources/MockBundle.swift

```diff
@@ -42,13 +43,6 @@ final class MockBundle {
             os_log("Loading request %@ from: %@", log: MockDuck.log, type: .debug, "\(request)", inputURL.path)
             targetURL = inputURL
             targetLoadingURL = loadingURL
-        } else if
```

Comment:
We should not load from recorder files when recording. This will enable us to record multiple responses for the same request.